src/AutoMapper/XpressionMapper/Extensions/MapperExtensions.cs

```diff
@@ -23,12 +23,6 @@ public static TDestDelegate MapExpression<TDestDelegate>(this IMapper mapper, La
             if (expression == null)
                 return default(TDestDelegate);
 
-            if (expression.GetType().GetGenericTypeDefinition() != typeof(Expression<>)

```

Comment:
This fails in .Net Core 2.0 and fixing it seemed too much work for parameter validation.